andela/wg-aces
# #159250149 Change author's field to foreign field

#### What does this PR do?
Changes the author field from a single field to a foreign field

#### Description of Task to be completed?
Changing the author’s field for the exercises table into foreign field from single field. It should be represented as a foreign key in that exercise table.

#### How should this be manually tested?
After cloning this repository;
- Login in the application and create a new exercise.
- Check the database the license author should be represented with a foreign key id in the exercises_exercise table.

#### What are the relevant pivotal tracker stories?
[#159250149](https://www.pivotaltracker.com/story/show/159250149)

#### Screenshots.
<img width="1199" alt="screen shot 2018-08-22 at 19 19 28" src="https://user-images.githubusercontent.com/39958516/44476458-74a6b400-a640-11e8-8e17-e35b2690576f.png">
